pkg/command/context.go

```diff
@@ -359,7 +359,7 @@ func globalLogin(c *configtypes.Context) (err error) {
 		issuer = csp.StgIssuer
 	}
 	if apiTokenExists {
-		log.Debug("API token env var is set")
+		log.Info("API token env var is set")
```

Comment:
This one is not on an error path and will print when it didn't before.
But maybe in this case it is actually good, as it might be interesting to the user.  I'm personally ok with keeping this one.  Maybe @prkalle has an opinon.
```
$ export TANZU_API_TOKEN=allo
$ tz context create --name tmc-unstable --endpoint unstable.tmc-dev.cloud.vmware.com --staging
API token env var is set
[...]
```

Reply:
I am okay with keeping, but
- if it is still somewhat debatable, maybe put it behind a low log level (<=3) instead of unconditionally showing
- would mentioning the env var name itself be more useful?